rust/abacus-core/src/types/chain_data.rs

```diff
@@ -0,0 +1,57 @@
+use ethers::prelude::{H256, U256};
+
+/// Info about a given block in the chain.
+#[derive(Debug)]
+pub struct BlockInfo {
```

Comment:
Which fields do we actually surface in the explorer? Why do we need to scrape gas_used and gas_limit?

Reply:
We fetch the data anyway, so we store it in the database, if it does not get used, oh well. But it is also there if we want to perform analysis later. This was one of those things that was trivial to add in addition to what we already needed to do.

Reply:
Personally I don't think these will ever be useful metrics at the block level and would prefer to remove

Reply:
Removed